cmd/tailscale/cli/serve_v2_test.go

```diff
@@ -745,6 +746,10 @@ func TestServeDevConfigMutations(t *testing.T) {
 		command: cmd("serve --https=4545 --bg --yes localhost:3000 off"),
 		want:    &ipn.ServeConfig{},
 	})
+	add(step{
+		command: cmd("funnel --http=10000 --bg 3000"),
+		wantErr: exactErrMsg(errors.New("error parsing commandline arguments: flag provided but not defined: -http")),
+	})
```

Comment:
might be worth writing it in terms of https://github.com/tailscale/tailscale/pull/9959